### PR TITLE
Thrift health checks in Hyperbahn

### PR DESCRIPTION
**Pull request by ShanniLi**
r @Raynos @kriskowal @anson627

### CODE REVIEW

File: hyperbahn/endpoints/index.js [hunk 1]
```diff
@@ -68,4 +70,12 @@ function setupEndpoints(clients, services) {
         //     handle(req, res, arg2, arg3);
         // }
     });
+
+    /*eslint no-unused-vars: 0*/
+    // thrift health endpoint
+    var tchannelAsThrift = new TChannelAsThrift({
+        source: '',
+        channel: clients.autobahnChannel,
```
Comment (Raynos) [inline hyperbahn/endpoints/index.js]: Let's use the `clients.hyperbahnChannel`

File: hyperbahn/test/lib/test-client.js [hunk 2]
```diff
@@ -113,6 +145,18 @@ TestClient.prototype.sendHealth = function sendHealth(cb) {
     }, cb);
 };
 
+TestClient.prototype.sendHealthThrift = function sendHealthThrift(cb) {
+    var self = this;
+    var healthSpec = fs.readFileSync(path.join(__dirname, '../../node_modules/tchannel/as/meta.thrift'), 'utf8');
+
+    thriftSend(self, {
+        endpoint: 'Meta::health',
+        serviceName: 'autobahn',
```
Comment (Raynos) [inline hyperbahn/test/lib/test-client.js]: lets send to `hyperbahn`

File: hyperbahn/endpoints/health.js [hunk 3]
```diff
@@ -29,3 +30,10 @@ function home(opts, req, head, body, cb) {
         body: 'hello from autobahn\n'
     });
 }
+
+function isHealthy() {
+    return {
+        ok: true,
+        message: 'hello from autobahn\n'
```
Comment (anson627) [inline hyperbahn/endpoints/health.js]: hyperbahn?

File: hyperbahn/endpoints/index.js [hunk 4]
```diff
@@ -68,4 +70,12 @@ function setupEndpoints(clients, services) {
         //     handle(req, res, arg2, arg3);
         // }
     });
+
+    /*eslint no-unused-vars: 0*/
+    // thrift health endpoint
+    var tchannelAsThrift = new TChannelAsThrift({
+        source: '',
```
Comment (anson627) [inline hyperbahn/endpoints/index.js]: empty source?

Comment (ShanniLi) [inline hyperbahn/endpoints/index.js]: Yes, that is something we have to live with for now ... until we have a real thrift IDL for Hyperbahn.

**Comment by Raynos** [comment]
legit.

**Pull request merged**